internal/mode/static/nginx/config/servers_template.go

```diff
@@ -51,6 +51,8 @@ server {
         proxy_set_header {{ $h.Name }} "{{ $h.Value }}";
             {{- end }}
         proxy_set_header Host $gw_api_compliant_host;
+        proxy_set_header Upgrade $http_upgrade;
```

Comment:
considering https://nginx.org/en/docs/http/websocket.html uses ` proxy_http_version 1.1;` , should we enable it too?

By default, NGINX uses HTTP/1.0 for backends.